### PR TITLE
- Introduced a new validation and updated files

### PR DESCRIPTION
**Pull request by Evanskorir**
- added two scaling options using elasticity
- including transition probabilities to link NGM as suggested in outcome_transition_params.py

### CODE REVIEW

File: src/models/rost/v_matrix_calculator.py [hunk 1]
```diff
@@ -44,4 +44,4 @@ def _get_v(self):
         # I3 -> Ih/Ic (& R)
         v[idx("i3"), idx("i3")] = self.n_i * self.parameters["gamma_s"]
 
-        self.v_inv = torch.linalg.inv(v)
+        self.v_inv = torch.linalg.inv(v)
```
Comment (zsvizi) [inline src/models/rost/v_matrix_calculator.py]: Revert this change.

File: src/static/contact_mtx_eigen.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import torch
```
Comment (zsvizi) [inline src/static/contact_mtx_eigen.py]: Sort alphabetically.

File: src/gradient/sensitivity_calculator.py [hunk 3]
```diff
@@ -1,21 +1,29 @@
 import src.models as models
```
Comment (zsvizi) [inline src/gradient/sensitivity_calculator.py:1]: Remove elasticity NGM code, which are not used now.
